boa_engine/src/vm/opcode/mod.rs

```diff
@@ -443,6 +529,14 @@ pub(crate) trait Operation {
     const INSTRUCTION: &'static str;
 
     fn execute(context: &mut Context<'_>) -> JsResult<CompletionType>;
+
+    fn half_execute(context: &mut Context<'_>) -> JsResult<CompletionType> {
+        Reserved::half_execute(context)
+    }
+
+    fn wide_execute(context: &mut Context<'_>) -> JsResult<CompletionType> {
+        Reserved::wide_execute(context)
+    }
```

Comment:
The default impls for half and wide just exist to enable implementing `Operation` without providing all three methods right? I think a comment noting that would be nice.